lib_xua/src/core/audiohub/xua_audiohub_st.h

```diff
@@ -1,88 +1,121 @@
 // Copyright 2011-2025 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 
-#pragma unsafe arrays
-static inline unsigned DoSampleTransfer(chanend ?c_out, const int readBuffNo, const unsigned underflowWord)
+#include "xua_commands.h"
+
+static inline void StartSampleTransfer(chanend ?c_aud, const unsigned underflowWord)
 {
-    if(XUA_USB_EN && ((NUM_USB_CHAN_OUT > 0) || (NUM_USB_CHAN_IN > 0)))
+    if(XUA_USB_EN)
```

Comment:
Worried that this might break a config that has USB Audio but USB for other reasons (such as HID) and I2S running. Such as the I2S config of an02016 (AudioWeaver)

Reply:
Noticed this is handled elsewhere now